terraform/deployments/mobile-backend/gha-iam-role.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  github_oidc_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
+}
```

Comment:
Is this being used anywhere?

Reply:
correct not being used anymore; have removed